shared/lib/sharing/action-types.js

```diff
@@ -0,0 +1,3 @@
+export const FETCH_PUBLICIZE_CONNECTIONS = 'FETCH_PUBLICIZE_CONNECTIONS';
+export const RECEIVE_PUBLICIZE_CONNECTIONS = 'RECEIVE_PUBLICIZE_CONNECTIONS';
+export const FAIL_PUBLICIZE_CONNECTIONS_REQUEST = 'FAIL_PUBLICIZE_CONNECTIONS_REQUEST';
```

Comment:
(My question makes more sense when picturing a lib that grows to have a few more actions that this one.)